src/api.js

```diff
@@ -21,4 +21,5 @@ export const api = {
   saveAnnotations: '/save_annotations/',
   getImageAnnotations: id => `/get_image_annotations/${ id }/`,
   getAccountDetails: id => axios.get(`get_user_info/${ id }`),
+  getUserAnnotations: id => axios.get(`http://localhost:8000/get_user_annotation_info/${ id }/`),
```

Comment:
@mbwatson Should ```http://localhost:8000``` be removed from the url in axios.get() above?

Reply:
haha yep!